v2/src/css/custom.css

```diff
@@ -519,3 +519,48 @@ html[data-theme="dark"] .markdown blockquote > *:last-child {
     bottom: 144px !important;
   }
 }
+
+/* Styles for collapsible elements */
+details {
+  position: relative;
+  transition: 0.3s, 0s border;
+  margin: 6px 0;
+}
+
+details > summary {
+  list-style-type: none;
+  position: relative;  
+  display: flex;
+  align-items: center;
+  background-color: var(--ifm-pre-background);  
+  padding: 11px 20px;
+  font-size: 16px;
+  cursor: pointer;
+}
+
+details > summary::before {
+  content: "";
+  position: relative;
+  box-sizing: border-box;
+  display: inline-block;
+  background: url(/img/icon-plus.svg) center;
```

Comment:
The svg appears different on different devices, use a png instead

Reply:
updated